internal/controllers/machinedeployment/machinedeployment_sync.go

```diff
@@ -236,13 +237,13 @@ func (r *Reconciler) getNewMachineSet(ctx context.Context, d *clusterv1.MachineD
 
 		return nil, err
 	case err != nil:
-		log.Error(err, "Failed to create new machine set", "machineset", newMS.Name)
+		log.Error(err, "Failed to create new MachineSet", "MachineSet", klog.KObj(&newMS))
 		r.recorder.Eventf(d, corev1.EventTypeWarning, "FailedCreate", "Failed to create MachineSet %q: %v", newMS.Name, err)
 		return nil, err
 	}
 
 	if !alreadyExists {
-		log.V(4).Info("Created new machine set", "machineset", createdMS.Name)
+		log.V(4).Info("Created new MachineSet", "MachineSet", klog.KObj(&newMS))
```

Comment:
```suggestion
		log.V(4).Info("Created new MachineSet", "MachineSet", klog.KObj(&createdMS))
```
Probably (?)

Reply:
It's the same at this point, right? I'll update as it might be more clear.

Reply:
Hm yeah, good point. Just saw the diff to before